docs/app/pages/charts/sections/scrollable-chart/scrollable-chart/scrollable-chart.component.html

```diff
@@ -0,0 +1,137 @@
+<div class="demo-chart-container">
+
+    <div class="chart-scroll-previous-btn" *ngIf="hasPreviousPage()" (click)="goToPreviousPage()">
+        <i class="hpe-icon hpe-previous"></i>
+    </div>
+
+    <canvas baseChart 
+            [datasets]="barChartData"
+            [labels]="barChartLabels"
+            [options]="barChartOptions"
+            [legend]="barChartLegend"
+            [colors]="barChartColors"
+            chartType="bar">
+    </canvas>
+
+    <div class="chart-scroll-next-btn" *ngIf="hasNextPage()" (click)="goToNextPage()">
+        <i class="hpe-icon hpe-next"></i>        
+    </div>
+
+</div>
+
+<hr>
+
+<p>
+    Scrollable Charts can be added using the <code>ng2-charts</code> library. 
+    The <code>chart.js</code> libray needs to be imported and the <code>ChartsModule</code> needs to be added to the appropriate NgModule.
+</p>
+
+<br>
+
+<p>The <code>baseChart</code> directive should be added to a <code>canvas</code> element and the following attributes can be used to customize the chart behavior and appearance:</p>
+
```

Comment:
Some more explanation might be needed here since scrolling isn't provided by the ng2-charts but rather provided through custom code.